src/Components/Telemetry.md

```diff
@@ -48,6 +48,29 @@ Aspire.Azure.Messaging.WebPubSub:
 - Metric names:
   - none (currently not supported by the Azure SDK)
 
+Aspire.Azure.Npgsql:
```

Comment:
I wonder if we could just add this name to the `Aspire.Npgsql` section, so we don't need to dupe here.